### PR TITLE
fix notebook bug (#362)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Fix Bug #362 
The problem is much simpler than I thought, the ipykernel used by vscode have the same --force-https argument that we have

### CODE REVIEW

File: src/taipy/gui/config.py [hunk 1]
```diff
@@ -170,7 +172,7 @@ def _handle_argparse(self, args):  # pragma: no cover
             config["use_reloader"] = False
         if args.ngrok_token:
             config["ngrok_token"] = args.ngrok_token
-        if args.force_https:
+        if args.use_https:
             config["force_https"] = True
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/config.py]: Should we change the config param from force_https to use_https everywhere for the sake of uniformity and doc ?

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]

**Review by FredLL-Avaiga — APPROVED** [review]
I guess that https will be provided via a proxy ?

**Comment by dinhlongviolin1** [comment]
> I guess that https will be provided via a proxy ?

Definitely

**Review by FabienLelaquais — APPROVED** [review]
Less is more.

**Pull request merged**